lib/addons/p5.dom.js

```diff
@@ -2692,10 +2692,10 @@
       }
 
       return p5.Renderer2D.prototype.get.call(this, x, y, w, h);
-    } else if (typeof x === 'undefined') {
+    } else if (typeof w === 'undefined' || typeof h === 'undefined') {
       return new p5.Image(1, 1);
     } else if (w > 1) {
```

Comment:
should this line be `(w >= 1)`? it seems in the case of `w === 1` it would return a black pixel regardless of x and y.

Reply:
i think actually this fix isn't relevant any more - there were a bunch of other changes in this area that happened. there are still bugs related to the pixels stuff, but i'm going to close this PR until i have a better handle on it.

something that we may want to consider, aside from the bugs, is that sometimes `get()` returns an `Image` and other times it returns a color value (in an array) which is fine, mostly. curiously, when requesting a sub-image that overlaps the source image boundary, a color array is returned:

https://github.com/processing/p5.js/blob/edb41cf513efafd3734221a3b8cdc266f90d6bb3/src/image/pixels.js#L502-L505

which seems a little strange: move one pixel to the left and suddenly the type of the object returned is completely different. i would expect that it just return an image with black/transparent pixels that come from outside the source image.